src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -39,6 +55,14 @@ const StyledGelPageGrid = styled(GelPageGrid)`
   flex-grow: 1; /* needed to ensure footer positions at bottom of viewport */
 `;
 
+const StyledWrapper = styled.div`
```

Comment:
We could (should?) remove the need for this wrapping div by allowing `VideoPlayer` to be styled directly, eg

`const StyledVideoPlayer = styled(VideoPlayer)`

This may need  updates to VideoPlayer to pass along the `className` prop:  https://styled-components.com/docs/basics#styling-any-component